packages/core/src/domain/session/sessionState.spec.ts

```diff
@@ -35,12 +63,16 @@ describe('session state utilities', () => {
     })
 
     it('should handle empty session strings', () => {
+      expect(toSessionState('')).toEqual({})
+    })
+
+    it('should handle expired session', () => {
       expect(toSessionState(SERIALIZED_EXPIRED_SESSION)).toEqual(EXPIRED_SESSION)
     })
 
     it('should handle invalid session strings', () => {
       const sessionString = '{invalid: true}'
-      expect(toSessionState(sessionString)).toEqual(EXPIRED_SESSION)
+      expect(toSessionState(sessionString)).toEqual({})
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about extracting `NOT_STARTED_SESSION` constant?